pkg/config/config.go

```diff
@@ -1003,6 +1003,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("external_metrics_provider.config", map[string]string{})        // list of options that can be used to configure the external metrics server
 	config.BindEnvAndSetDefault("external_metrics_provider.local_copy_refresh_rate", 30)        // value in seconds
 	config.BindEnvAndSetDefault("external_metrics_provider.chunk_size", 35)                     // Maximum number of queries to batch when querying Datadog.
+	config.BindEnvAndSetDefault("external_metrics_provider.split_batches_and_backoff", false)   // Splits batches and runs queries with errors individually with an expnential backoff
```

Comment:
```suggestion
	config.BindEnvAndSetDefault("external_metrics_provider.split_batches_and_backoff", false)   // Splits batches and runs queries with errors individually with an exponential backoff
```

Reply:
I'd probably also lean towards `split_batches_with_backoff` but not a blocker

Reply:
Thanks for suggestion, renamed!